src/main/java/mil/dds/anet/resources/ReportResource.java

```diff
@@ -152,6 +151,8 @@ public Report createReport(@GraphQLRootContext Map<String, Object> context,
     r.setReportText(
         Utils.isEmptyHtml(r.getReportText()) ? null : Utils.sanitizeHtml(r.getReportText()));
 
+    ResourceUtils.assertAllowedClassification(r.getClassification());
```

Comment:
I would move this assertion up a bit, like directly after checking the report state.